source/clear-linux/tutorials/aws-web/aws-web.rst

```diff
@@ -0,0 +1,266 @@
+.. _aws-web:
+
+Create and launch Clear Linux from Amazon Web Services
+######################################################
+
+This tutorial walks you through creating and launching a |CL|
+:abbr:`AMI (Amazon Machine Image)` instance from the
+:abbr:`AWS (Amazon Web Services)` console and complete the following tasks:
+
+#. Locate and select the |CL| OS Basic AMI in the AWS Marketplace.
+#. Create a new public and private key pair to allow you to connect to your
+   |CL| instance securely.
+#. Launch the new |CL| instance and connect to it.
+#. Update your instance of |CL| using the :command:`swupd` command.
+#. Stop the |CL| instance.
+
+Prerequisites
+*************
+
+This tutorial assumes the following:
```

Comment:
This tutorial assumes the following statements are true:

Reply:
done